app/src/main/java/com/keylesspalace/tusky/fragment/preference/AccountPreferencesFragment.kt

```diff
@@ -57,19 +58,19 @@ class AccountPreferencesFragment : PreferenceFragmentCompat(),
     @Inject
     lateinit var eventHub: EventHub
 
-    private lateinit var notificationPreference: Preference
-    private lateinit var tabPreference: Preference
-    private lateinit var mutedUsersPreference: Preference
-    private lateinit var blockedUsersPreference: Preference
+    private var notificationPreference: Preference? = null
```

Comment:
I wouldn't do that tbh. They should be there, we shouldn't check for them in general.

Reply:
Yes, I agree with @charlag, it should crash when the preference is not found. Since it is used a lot and to keep the code clean I would add an extension function `requirePreference` that returns a non-null preference or throws a exception.